ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2512,6 +2514,15 @@ def test074_create_existing_order_change_contact(self):
         self.order_page.create_existing_order_with_auto_fill(no=order_no_with_year)
         self.order_page.sleep_tiny()
         new_contact = self.order_page.set_contact(remove_old=True)[0]
```

Comment:
don't set contact to random contact, you need to set it to contact with department ( not in original order)
`        contacts_list = ContactsAPI().get_contacts_with_department()
`
then remove original contact from it, then select one random
Or, you can create new contact with department:
`        response, payload = ContactsAPI().create_contact_with_multiple_departments()
`
I prefer second solution .